wear/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+  <string name="hello_world">Hello Wear World!</string>
```

Comment:
A heads up for future PRs: Each feature has its own `resources.xml` file.

For example, if "hello world" was a feature it would have a file called `hello_world-resources.xml` (use `_` as spaces, and `-` for the file type post fix)